airflow/migrations/versions/37c6b55d15a9_merge_migration_heads.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""merge migration heads
+
+Revision ID: 37c6b55d15a9
+Revises: 74effc47d867, 004c1210f153
+Create Date: 2019-10-02 16:40:50.585970
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '37c6b55d15a9'
+down_revision = ('74effc47d867', '004c1210f153')
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Apply merge migration heads"""
+    pass
+
+
+def downgrade():
+    """Unapply merge migration heads"""
+    pass
```

Comment:
Had 2 heads so I merged them before creating my migration.

Reply:
I think this has been dealt with on master already -- could you rebase and remove this if there are not two heads anymore?

Reply:
> I think this has been dealt with on master already -- could you rebase and remove this if there are not two heads anymore?

Have updated with latest master, removed the heads merge commit, updated migration down revision and squashed linting fixes into the main commit 👍